src/examples/pinball/pinballflowsolver.py

```diff
@@ -267,6 +269,21 @@ def _make_BCs(self):
 
         return BC
 
+    def _default_steady_state_initial_guess(self) -> dolfin.UserExpression:
```

Comment:
Suggestion: 
- use `FlowSolver.compute_steady_state(..., initial_guess=custom_initial_guess())` 
- where you define `custom_initial_guess()` as a subclass of `dolfin.UserExpression` (same as the `default_initial_guess` in `FlowSolver_default_steady_state_initial_guess(self)`)

This would avoid commenting/uncommenting from here

Reply:
I think this way is actually easier if we make it work - I will look into it soon, I don't think it should be a big problem. It also has the good property of showing different ways of doing the same thing (cavity redefines default, pinball introduces a class...). We will see how it goes!